File: codewars/python/total_amount_points.py
def points(games):
    total_score  =0
    for game in games:
        if(game[0] > game[-1]):
            total_score+=3
        elif(game[0] == game[-1]):
            total_score+=1
    return total_score             

print(points(['1:0','2:0','3:0','4:0','2:1','3:1','4:1','3:2','4:2','4:3'])) #30
print(points(['1:1','2:2','3:3','4:4','2:2','3:3','4:4','3:3','4:4','4:4'])) #10

File: codewars/c/total_amount_points.c
#include <unistd.h>
int points(const char *const games[])
{
  int total_points = 0;
  for (int i = 0; i < 10; i++)
  {
    int t1 = atoi(games[i][0]);
    int t2 = atoi(games[i][2]);
    if (t1 > t2)
    {
      total_points += 3;
    }
    else if (t1 == t2)
    {
      total_points++;
    }
  }
  return total_points;
}

int main()
{
  return 0;
}